cabal.project

```diff
@@ -32,11 +32,14 @@ package bitvec
   flags: -simd
 
 extra-packages: Cabal
+
 if os(windows)
   constraints: time ^>=1.14
   allow-newer: *:time
 
-constraints: any.text source
+-- Temporary workaround for problem with cabal-doctest
```

Comment:
Points for the comment :+1:. Which repo is having issues with cabal-doctest?

Reply:
Ah that is from @neilmayhew's commits. Mine is just the last one. But I think it is consensus the one that had this problem. https://github.com/IntersectMBO/ouroboros-consensus/blob/main/cabal.project#L48-L49

Reply:
I tested this, and the constraint can be taken out in consensus, api and cli. However, it can't be taken out in node because it depends (indirectly) on `entropy` which uses a custom `Setup.hs` that isn't compatible with `Cabal-3.14` when built with `ghc8107`.

Reply:
I've pushed a commit to my branch in cli that removes the constraint. You can just rebase on that.

Reply:
I've discovered that the problem can be worked around more cleanly in node by updating the Hackage index, because `entropy` was updated to be compatible with `Cabal-3.14` on Jan 1.